llvm/docs/LangRef.rst

```diff
@@ -14416,7 +14416,7 @@ Syntax:
 ::
 
       declare void @llvm.instrprof.mcdc.tvbitmap.update(ptr <name>, i64 <hash>,
-                                                        i32 <bitmap-bytes>)
+                                                        i32 <unused>)
```

Comment:
I saw a pattern across whole PR where you replace bytes with bits ... is it intentional here to have the argument unused instead of something like <bitmap-bits> ?

Reply:
I wanted to leave the signature of instrinsics as-is. I will remove it in the next request.
(FYI, condupdate is not used)